### DISCUSSION TITLE
Velvet in Ipmanchess list

### DISCUSSION THREAD

**Discussion by Ipmanchess** [mhonert/velvet-chess#8] [Show and tell]
Hi..

Don't know if you are on Discord..but i'm already a time testing your engine Velvet !

Here you can see your progress in this list: [https://ipmanchess.yolasite.com/i7-11800h.php](url)
scroll also more down for older versions

Sorry that i put it here as a issue ,it was just a way to contact you..

Kind regards,
Ipman.

**Comment by mhonert** [comment]
Hi, thanks for testing Velvet!
I converted the issue into a discussion entry.
